ui/package.json

```diff
@@ -15,9 +15,13 @@
         "test:ci": "yarn -s test -- --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
         "clean:dist": "rm -rf ./dist",
         "compile": "BABEL_ENV=production yarn -s install && yarn -s clean:dist && webpack --config ./__config__/webpack.config.prod.js --progress && echo 'Client bundle size:' && gzip-size ../static/target/javascripts/ui.bundle.browser.js",
+        "compile:dev": "yarn compile:dev:server && yarn compile:dev:browser",
```

Comment:
as discussed IRL, maybe we should stick with `make ui-watch` (this change should be invisible to devs?) but add `make ui-watch-nashorn` for situations we want that?

Reply:
where does `compile:dev` get called?

Reply:
`compile:dev` and `compile:dev:browser` don't get called at the moment. I included them for  completionist / aesthetic / anally retentive reasons, but I can probably get rid of them 😅